packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -0,0 +1,451 @@
+    category: 'signaling',
+    fatal: true,
+    name: 'locus.response',
+  },
+  1006: {
+    errorDescription: ERROR_DESCRIPTIONS.LOCUS_INVALID_SEQUENCE_HASH,
+    category: 'signaling',
+    fatal: true,
+  },
+  1007: {
+    errorDescription: ERROR_DESCRIPTIONS.UPDATE_MEDIA_FAILED,
+    category: 'signaling',
+    fatal: true,
+  },
+  2001: {
+    errorDescription: ERROR_DESCRIPTIONS.FAILED_TO_CONNECT_MEDIA,
+    category: 'signaling',
+    fatal: true,
+  },
+  2002: {
+    errorDescription: ERROR_DESCRIPTIONS.MEDIA_ENGINE_LOST,
+    category: 'signaling',
+    fatal: true,
+  },
+  2003: {
+    errorDescription: ERROR_DESCRIPTIONS.MEDIA_CONNECTION_LOST,
+    category: 'signaling',
+    fatal: true,
+  },
+  2004: {
+    errorDescription: ERROR_DESCRIPTIONS.ICE_FAILURE,
+    category: 'signaling',
+    fatal: true,
+  },
+  2005: {
+    errorDescription: ERROR_DESCRIPTIONS.MEDIA_ENGINE_HANG,
+    category: 'signaling',
+    fatal: true,
+  },
+  2006: {
+    errorDescription: ERROR_DESCRIPTIONS.ICE_SERVER_REJECTED,
+    category: 'signaling',
+    fatal: true,
+  },
+  3001: {
+    errorDescription: ERROR_DESCRIPTIONS.CALL_FULL,
+    category: 'expected',
+    fatal: true,
+  },
+  3002: {
+    errorDescription: ERROR_DESCRIPTIONS.ROOM_TOO_LARGE,
+    category: 'expected',
+    fatal: true,
+  },
+  3004: {
+    errorDescription: ERROR_DESCRIPTIONS.GUEST_ALREADY_ADDED,
+    category: 'expected',
+    fatal: false,
+  },
+  3005: {
+    errorDescription: ERROR_DESCRIPTIONS.LOCUS_USER_NOT_AUTHORISED,
+    category: 'expected',
+    fatal: true,
+  },
+  3006: {
+    errorDescription: ERROR_DESCRIPTIONS.CLOUDBERRY_UNAVAILABLE,
+    category: 'expected',
+    fatal: true,
+  },
+  3007: {
+    errorDescription: ERROR_DESCRIPTIONS.ROOM_TOO_LARGE_FREE_ACCOUNT,
+    category: 'expected',
+    fatal: true,
+  },
+  4001: {
+    errorDescription: ERROR_DESCRIPTIONS.MEETING_INACTIVE,
+    category: 'expected',
+    fatal: true,
+  },
+  4002: {
+    errorDescription: ERROR_DESCRIPTIONS.MEETING_LOCKED,
+    category: 'expected',
+    fatal: true,
+    name: 'locus.response',
+  },
+  4003: {
+    errorDescription: ERROR_DESCRIPTIONS.MEETING_TERMINATING,
+    category: 'expected',
+    fatal: true,
+    name: 'locus.leave',
+  },
+  4004: {
+    errorDescription: ERROR_DESCRIPTIONS.MODERATOR_PIN_OR_GUEST_REQUIRED,
+    category: 'expected',
+    fatal: false,
+  },
+  4005: {
+    errorDescription: ERROR_DESCRIPTIONS.MODERATOR_PIN_OR_GUEST_PIN_REQUIRED,
+    category: 'expected',
+    fatal: false,
+  },
+  4006: {
+    errorDescription: ERROR_DESCRIPTIONS.MODERATOR_REQUIRED,
+    category: 'expected',
+    fatal: false,
+  },
+  4007: {
+    errorDescription: ERROR_DESCRIPTIONS.USER_NOT_MEMBER_OF_ROOM,
+    category: 'expected',
+    fatal: true,
+  },
+  4008: {
+    errorDescription: ERROR_DESCRIPTIONS.NEW_LOCUS_ERROR,
+    category: 'expected',
+    fatal: true,
+  },
+  4009: {
+    errorDescription: ERROR_DESCRIPTIONS.NET_WORK_UNAVAILABLE,
```

Comment:
Can we change this to `NETWORK_UNAVAILABLE`?

Reply:
fixed